vendor/engines/bulk_email/app/assets/javascripts/bulk_email/bulk_email.js

```diff
@@ -1,9 +1,19 @@
 $(function() {
+
+  function selected_user_types() {
```

Comment:
I _think_ we've been moving towards using camel case for JS function and variable names.